src/Application/GameStarter.cpp

```diff
@@ -40,14 +41,18 @@
 #include "GameKeyboardController.h"
 #include "GameWindowHandler.h"
 #include "GameTraceHandler.h"
+#include "GameLuaBindings.h"
 
 GameStarter::GameStarter(GameStarterOptions options): _options(std::move(options)) {
     // Init environment.
     _environment = Environment::createStandardEnvironment();
 
     // Init logger.
     _bufferLogSink = std::make_unique<BufferLogSink>();
-    _defaultLogSink = LogSink::createDefaultSink();
+    auto compositeLogSink = std::make_unique<LogSinkComposite>();
+    auto compositeLogSinkPtr = compositeLogSink.get();
+    compositeLogSink->addLogSink(LogSink::createDefaultSink());
+    _defaultLogSink = std::move(compositeLogSink);
```

Comment:
Just change the type of `_defaultLogSink` to `LogSinkComposite`, then you won't need to store `compositeLogSinkPtr`.